src/webgl/p5.RendererGL.Retained.js

```diff
@@ -336,4 +336,31 @@ p5.RendererGL.prototype._drawElements = function(drawMode, gId) {
   this._pInst._pixelsDirty = true;
 };
 
+p5.RendererGL.prototype.initPointVertexBuffer = function(x, y, z) {
+  var gl = this.GL;
+
+  var pointVertexBuffer = gl.createBuffer();
+
+  this._bindBuffer(
+    pointVertexBuffer,
+    gl.ARRAY_BUFFER,
+    [x, y, z],
```

Comment:
Hmm, seeing this, I wonder if some slight performance gains can be made by making an array once and keeping it around.